spec/models/auction_status_spec.rb

```diff
@@ -1,155 +1,115 @@
-require "rails_helper"
+require 'rails_helper'
 
 describe AuctionStatus do
-  describe "#available?" do
-    context "start datetime in past, end datetime in future" do
-      it "is true" do
-        auction = FactoryGirl.build(:auction, started_at: yesterday, ended_at: tomorrow)
-
+  describe '#available?' do
+    context 'start datetime in past, end datetime in future' do
+      it 'is true' do
+        auction = FactoryGirl.build(:auction, :available)
         status = AuctionStatus.new(auction)
-
         expect(status).to be_available
       end
-
-      context "start datetime in past, end datetime in past" do
-        it "is false" do
-          auction = FactoryGirl.build(:auction, started_at: yesterday, ended_at: yesterday)
-
-          status = AuctionStatus.new(auction)
-
-          expect(status).not_to be_available
-        end
-      end
-
-      context "start datetime in future, end datetime in future" do
-        it "is false" do
-          auction = FactoryGirl.build(:auction, started_at: tomorrow, ended_at: tomorrow)
-
-          status = AuctionStatus.new(auction)
-
-          expect(status).not_to be_available
-        end
-      end
     end
 
-    describe "#over?" do
-      context "end datetime is in past" do
-        it "is true" do
-          auction = FactoryGirl.build(:auction, ended_at: yesterday)
-
-          status = AuctionStatus.new(auction)
-
-          expect(status).to be_over
-        end
+    context 'start datetime in past, end datetime in past' do
+      it 'is false' do
+        auction = FactoryGirl.build(:auction, :closed)
```

Comment:
thoughts on *not* using `FactoryGirl` prepend thingy? Seems like we are doing this inconsistently around our test suite.